app/src/main/res/layout-sw600dp-land/welcome.xml

```diff
@@ -42,13 +42,28 @@
       app:layout_constraintBottom_toBottomOf="@id/welcome_text_view"
       app:layout_constraintTop_toTopOf="@+id/welcome_text_view" />
 
+    <TextView
+      android:id="@+id/profile_name_textview2"
+      android:layout_width="0dp"
+      android:layout_height="wrap_content"
+      android:ellipsize="end"
+      android:fontFamily="sans-serif"
+      android:maxLines="1"
```

Comment:
For textview 2 we should remove `maxLines` and `ellipsize` so that the name is fully visible. This change should be done in all layouts.

Reply:
Done